### PR TITLE
adding dev resources to docs

### PR DESCRIPTION
**Pull request by sodofi**
these are subjective changes. please review before merging.

### CODE REVIEW

File: docs/build/add-celo-testnet-to-metamask.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+2. Go to settings
+
+![](/img/build/add-celo-testnet-to-metamask/2.png)
+
+3. Select Networks
+
+![](/img/build/add-celo-testnet-to-metamask/3.png)
+
+4. Add info about Celo L2 Dango Testnet
```
Comment (aaronmgdr) [inline docs/build/add-celo-testnet-to-metamask.md:19]:
```suggestion
4. Add info about Celo L2 Alfajores Testnet
```

File: docs/build/build-on-minipay/code-library.mdx [hunk 2]
```diff
@@ -0,0 +1,620 @@
...
+
+To use the code snippets below, install the following packages:
+
+{/* prettier-ignore-start */}
+
+<Tabs>
+  <TabItem value="yarn" label="yarn" attributes={{ className: "yarn_tab" }} default>
+
+        ```bash
+        yarn add @celo/abis @celo/identity viem@1
```
Comment (aaronmgdr) [inline docs/build/build-on-minipay/code-library.mdx:73]: viem is on version 2 now. is 1 required?

File: docs/general/index.md [hunk 3]
```diff
@@ -1,4 +1,77 @@
...
+- **Fee Abstraction**: Users can pay transaction fees with several different tokens, making payments simple and flexible.
+- **Sub-Cent Fees**: Celo maintains low gas fees, often below a cent, keeping transactions affordable.
+- **Native Stablecoins**: Celo provides native stablecoins like cUSD, cEUR, cREAL, and cKES, offering a stable way to send and receive money. Check out [Mento](https://www.mento.org/) to learn more.
+
+
+#### Mobile-First Approach:
+
+Celo is optimized for mobile devices, making blockchain accessible to billions of smartphone users worldwide.
+
+- **Ultralight Client**: Celo’s lightweight client allows fast transactions on mobile phones without needing to download large amounts of data, ideal for areas with limited internet.
```
Comment (aaronmgdr) [inline docs/general/index.md:36]: Pretty sure this was deprecated.  @celo-org/blockchain

File: docs/general/ecosystem/guides/guidelines.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+```md
+---
+title: Contributing to the Blog
+description: How to contribute to the blog
+slug: blog-contributions
+authors:
+  - name: Josh Crites
+    title: Developer Relations, cLabs
+    url: https://github.com/critesjosh
```
Comment (aaronmgdr) [inline docs/general/ecosystem/guides/guidelines.md:134]: not sure it matters but do we want josh name to still be here?

File: docs/build/build-on-minipay/code-library.mdx (hunk 2 above)
Comment (aaronmgdr) [reply docs/build/build-on-minipay/code-library.mdx:73]: above it does say 2

File: docs/developer/explorers/block-explorers.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+title: Block Explorer
+---
+import ColoredText from '/src/components/ColoredText';
+
+
+
+## [Blockscout](https://explorer.celo.org)
```
Comment (aaronmgdr) [inline docs/developer/explorers/block-explorers.md:8]:
```suggestion
## [Blockscout](https://celo.blockscout.com)

File: docs/developer/explorers/block-explorers.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+Blockscout explorer is available for <ColoredText>[Celo](https://explorer.celo.org/mainnet/blocks)</ColoredText>. If your contract is verfied on Blockscout, you can interact with and debug smart contracts.
```
Comment (aaronmgdr) [inline docs/developer/explorers/block-explorers.md:10]: https://celo.blockscout.com/

File: docs/general/ecosystem/guides/guidelines.md (hunk 4 above)
Comment (ericnakagawa) [reply docs/general/ecosystem/guides/guidelines.md]: I've updated this with Octocat placeholder.

**Pull request closed without merging**